dashboard/app/jobs_test.go

```diff
@@ -350,3 +351,42 @@ func TestJobRestrictedManager(t *testing.T) {
 	c.expectEQ(pollResp.Manager, build.Manager)
 	c.expectEQ(pollResp.KernelRepo, "git://restricted.git/restricted.git")
 }
+
+// Test that BisectFixJob is returned only after 30 days
+func TestBisectFixJob(t *testing.T) {
+	c := NewCtx(t)
+	defer c.Close()
+
+	// Upload a crash report as reported 29 days ago and ensure that there
+	// is only 1 BisectCause job.
+	c.mockedTime = time.Now().Add(-24*29*time.Hour)
```

Comment:
Use `c.advanceTime` instead. Tests are now supposed to mess with c.mockedTime directly.
advanceTime is quite handy, it allows to test what happens when precisely. E.g.:
```
... ensure no bisect fix jobs created ...
c.advanceTime(31 * 24 * time.Hour)
... ensure a bisect fix job is created now ...
```

Reply:
I see, thanks, fixed